mvpa2/base/node.py

```diff
@@ -281,7 +281,9 @@ def set_postproc(self, node):
     def __str__(self, *args, **kwargs):
         return _str(self, *args, **kwargs)
 
-    def __repr__(self, prefixes=[]):
+    def __repr__(self, prefixes=None):
+        if prefixes is None:
+            prefixes = []
```

Comment:
I have no trust in function signatures. Moreover, the ones with all Nones in it defaults confuse me often